include/teqp/models/multifluid_ecs_mutant.hpp

```diff
@@ -0,0 +1,139 @@
+#pragma once
+#include <Eigen/Dense>
+
+    // Implementation of the polynomial extended corresponding states (pECS) mixture model
+    // The mixture model depends on coefficients for a temperature and density polynomial
+    // So far only valid for a binary mixture
+
+namespace teqp {
+
+    class Reducing_ECS {
```

Comment:
Might want to add some doxygen docs showing the mathematical form of the reducing function. Once you do, you should be able to check out the doc build here: https://github.com/usnistgov/teqp/actions/runs/8138577176